src/lib.rs

```diff
@@ -272,6 +275,12 @@ impl Flatc {
             cmd_args.push("--json".into());
         }
 
+        if !args.extra.is_empty() {
+            for extra_arg in args.extra {
+                cmd_args.push(extra_arg.into());
+            }
+        }
```

Comment:
If the list is empty, for loop will just do nothing

```suggestion
        for extra_arg in args.extra {
            cmd_args.push(extra_arg.into());
        }
```